docs/QUICK_START.md

```diff
@@ -14,14 +14,12 @@ All of the connections will be made to the back panel. Here's a quick reference
 - **POWER**: 115V, or optionally 230V connection, 115/230 switch is set on internal power supply and marked on rear of unit, see [Power](#power)
 - **CONTROLLER**: Connections to the embedded Raspberry Pi Controller
   - **SERVICE**: USB mini connection for re-imaging the Pi's EMMC
-  - **USB**:  USB A ports for connecting peripherals such as additional storage devices
-  - **OPTICAL IN**: SPDIF audio input, planned to be used for extra inputs
-  - **AUX IN**: Additional stereo input, planned to be used for announcements
-  - **HDMI OUT**: The Pi's HMDI output, can be used for visualizations or development
+  - **USB**:  USB A ports for connecting peripherals such as additional storage devices. **DO NOT USE AS A USB POWER SOURCE TO AN INPUT**
```

Comment:
Clarify this a bit. something like do not use as a power source for a device providing an audio output connected to any of the RCA inputs?